.github/workflows/cdk_package_code.yml

```diff
@@ -60,7 +61,7 @@ jobs:
             package-lock.json \
             tsconfig.defaults.json \
             Makefile \
-            cdk.json
+            cdk.json 
```

Comment:
Unnecessary whitespace at the end of line 64.

Reply:
fixed